src/components/PersonLink/PersonLink.tsx

```diff
@@ -0,0 +1,37 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { Person } from '../../types/Person';
+
+type Props = {
+  parent: string | null
+  getPersonByParent: (value: string | null) => Person | undefined
+};
+
+export const PersonLink: React.FC<Props> = ({
+  parent,
+  getPersonByParent,
+}) => {
+  const personIsParent = getPersonByParent(parent);
+
+  if (!parent) {
+    return (
+      <td> - </td>
+    );
+  }
+
+  return (
+    <td>
+      {personIsParent ? (
+        <Link
+          className={classNames({
+            'has-text-danger': personIsParent?.sex === 'f',
+          })}
+          to={`/people/${personIsParent?.slug}`}
+        >
+          {parent}
+        </Link>
+      ) : parent}
```

Comment:
```suggestion
      ) :  (
        {parent}
      )}
```

Reply:
When I put this variable into curly braces than I've got such an error:

Type 'Element | { parent: string; }' is not assignable to type 'ReactNode'.
  Type '{ parent: string; }' is not assignable to type 'ReactNode'.
    Object literal may only specify known properties, and 'parent' does not exist in type 'ReactElement<any, string | JSXElementConstructor<any>> | ReactFragment | ReactPortal'.ts(2322)